iosApp/iosApp/Explore/RepoListView.swift

```diff
@@ -7,23 +7,68 @@
 //
 
 import SwiftUI
+import ExploreShared
 
 struct RepoListView: View {
     @StateObject var viewModel = RepoListViewModel()
 
     var body: some View {
-        ForEach(self.viewModel.repoList, id: \.name) { repo in
-            VStack {
-                Text(repo.name)
-            }.padding(.bottom, 10)
-        }.onAppear {
-            Task {
-                await self.viewModel.loadRepos()
+        let list = self.viewModel.repoList
+        let isLastPage = self.viewModel.isLastPage
+
+        ZStack {
+            ScrollView(.vertical) {
+                LazyVStack {
+                    if !list.isEmpty {
+                        ListView(list)
+                    }
+                    
+                    if !isLastPage {
+                        LoadingView()
+                            .onAppear {
+                                Task {
+                                    await self.viewModel.loadRepos()
+                                }
+                            }
+                    }
+                }
             }
         }
     }
 }
 
+private struct ListView: View {
+    private var list: [Repo]
+    
+    init(_ list: [Repo]) {
```

Comment:
Do we really need to set the access level of the list var to private? Structs are immutable by default in Swift, and we usually don't try to access props of subviews inside the main view like `ListView(list).list = something`. By making the list public we also don't need to override the init.